ProjectK.Business/ExperiencePoints/Experience.cs

```diff
@@ -0,0 +1,61 @@
+﻿using ProjectK.Contracts;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    public interface IExperienceEdit
+    {
+        IExperienceEdit ForChar(int charId);
+        IExperienceEdit WithXpToLevel(ExperienceDto experience);
+    }
+    public class Experience : IExperienceEdit
+    {
+        [Key]
```

Comment:
Why? You should specify this in entity model builder.

Reply:
It is specified in entity model builder

Reply:
Then you don't need it here :D